firefox-ios/Client/Frontend/Browser/MainMenuActionHelper.swift

```diff
@@ -669,10 +670,10 @@ final class MainMenuActionHelper: PhotonActionSheetProtocol,
     /// non-HTML MIME type currently opened in the webView, such as a PDF).
     /// NOTE: Called from getShareFileAction (files in the browser) and getShareAction (websites)
     @MainActor
-    private func share(fileURL: URL, buttonView: UIView) {
+    private func share(fileURL: URL, remoteURL: URL?, buttonView: UIView) {
         TelemetryWrapper.recordEvent(category: .action, method: .tap, object: .sharePageWith)
         navigationHandler?.showShareSheet(
-            shareType: .file(url: fileURL),
+            shareType: .file(url: fileURL, remoteURL: remoteURL),
```

Comment:
instead of passing in `nil`, could we set `remoteURL: nil` and add the comment here instead. Unless there is an existing case in which we want the value to not be nil?